node.go

```diff
@@ -49,15 +52,25 @@ type netPayload struct {
 }
 
 // NewNode return a Node
-func NewNode(name string, style code.StyleCode, ip net.IP, log Logger) *Node {
+func NewNode(name string, style code.StyleCode, ip net.IP, log Logger, opts ...NodeOption) *Node {
 	n := &Node{
 		Config: NodeConfig{
 			Name: name,
 			Type: style,
 		},
-		conn:     nil,
-		shutdown: true,
-		log:      log.With(Fields{"type": "Node"}),
+		listenAddr:    net.UDPAddr{Port: packet.ArtNetPort},
+		broadcastAddr: defaultBroadcastAddr,
+		conn:          nil,
+		shutdown:      true,
+		log:           log.With(Fields{"type": "Node"}),
+	}
+
+	for _, opt := range opts {
+		n.SetOption(opt)
+	}
+
+	for _, opt := range opts {
```

Comment:
Why is this code block duplicated?

Reply:
Good catch. Must have slipped in while merging. I don't quite know why it thought that was a merge issue on that line... I'll fix it.